velox/docs/functions/spark/string.rst

```diff
@@ -50,6 +50,34 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
 
         SELECT replace('ABCabc', 'abc', 'DEF'); -- ABCDEF
 
+.. spark:function:: ltrim(string) -> varchar
```

Comment:
@izchen Just noticed that ltrim needs to be moved to preserve alphabetical order.